docs/releases/4.1.txt

```diff
@@ -329,6 +330,8 @@ Migrations
   ``AlterIndexTogether`` and ``AddIndex``, when moving indexes defined in the
   ``Meta.index_together`` to the
   :attr:`Meta.indexes <django.db.models.Options.indexes>`.
+* Squashed migrations can now themselves be squashed before being transitioned to
+  normal migrations.
```

Comment:
Thanks for picking up the baton, just strolled by to remind you to move this release note to 5.2.txt.

Reply:
Thanks for noticing, moved.